app/models/user.rb

```diff
@@ -7,6 +7,7 @@ class User < ActiveRecord::Base
   validates :name, presence: true
   validates :email, presence: true, uniqueness: true
   validates :password, presence: true, on: :create, length: { minimum: 7 }
+  before_create :generate_token
```

Comment:
Rather than generating token at user creation, I would recommend to generate it once user requests it. Otherwise its a huge security threat as token is not encrypted in your database as password is. Whoever steals the db can take control over the accounts with token generated.